.github/workflows/scripts/pre-submit.actions/references.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Verify that Action calls that are pinned (not locally called)
+# use the same referemnce as the relealse tag.
+# See reasoning in ./github/actions/README.md
+
+set -euo pipefail
+
+results=$(
+    find .github/workflows/ -name '*.yaml' -o -name '*.yml' \
```

Comment:
nit: We'll probably block on warnings at some point so ideally fix the shellcheck warning.

Reply:
I tried to but the shell run was complaining about invalid command :/

Reply:
I think you should be able to use `find ... -print0 | xargs -0 ...`. Did that not work for you? Maybe it's a flag not available on MacOS or something?

Reply:
mistake from my end. Done.